pkg/build_zip.py

```diff
@@ -0,0 +1,112 @@
+# Copyright 2015 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This tool builds zip files from a list of inputs."""
+
+from datetime import datetime
+from helpers import SplitNameValuePairAtSeparator
+from optparse import OptionParser
+from zipfile import ZipFile, ZipInfo, ZIP_DEFLATED
+
+ZIP_EPOCH = 315532800
+
+def get_options_parser():
+  parser = OptionParser()
```

Comment:
parser = argparse.ArgumentParser(fromfile_prefix_chars='@')

You fromfile_prefix_chars='@' for args.use_param_file to work in the pkg_zip rule.
https://docs.python.org/2/library/argparse.html#fromfile-prefix-chars